src/main/java/com/example/sixthmafiabot/exceptions/ValidationException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.sixthmafiabot.exceptions;
+
+import com.example.sixthmafiabot.exceptions.Abstract.BaseCustomException;
+import com.example.sixthmafiabot.exceptions.Abstract.CustomMapException;
+
+import java.util.Map;
+
+public class ValidationException extends CustomMapException {
```

Comment:
why do we need this if we have SerivceValidationException

Reply:
+

Reply:
?